src/modules/Host.rb

```diff
@@ -111,14 +111,14 @@ def Read
       return true if @initialized
 
       # read /etc/hosts
-      if Ops.greater_than(SCR.Read(path(".target.size"), CFA::Hosts::PATH), 0)
+      if File.exist?(CFA::Hosts::PATH)
```

Comment:
true ... but AFAIK not needed. Configured /etc/hosts is copied to the target at the end of installation.